File: 백준/Bronze/1978. 소수 찾기/소수 찾기.c
#include <stdio.h>

int main()
{
    int N, num, answer = 0;
    scanf("%d", &N);
    
    for (int i = 0; i < N; i++) {
        scanf("%d", &num);
        
        for (int j = 2; j <= num; j++) {
            if (num == j) answer++;
            
            if (num % j == 0) break;
        }
    }
    
    printf("%d", answer);
}

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
def is_prime(n):
    i = 5

    if n % 2 == 0:
        return n == 2

    if n % 3 == 0:
        return n == 3

    while i * i <= n:
        if n % i == 0:
            return False

        i += 2

    return n != 1

N, answer = int(input()), 0

for num in map(int, input().split()):
    if is_prime(num):
        answer += 1

print(answer)